troposphere/static/js/components/admin/Emulate.jsx

```diff
@@ -0,0 +1,41 @@
+import $ from "jquery";
+import React from "react";
+import ReactDOM from "react-dom";
+
+import Glyphicon from 'components/common/Glyphicon';
+import Tooltip from "components/common/ui/Tooltip";
```

Comment:
It looks like resource-requests is one of the only places it's being used. It's kind of hacky https://github.com/wwayne/react-tooltip. I guess you're supposed to place <ReactTooltip /> somewhere once in your dom, and then you can create instances of it with a separate tag. Not sure what is worse.